docs/Usage.md

```diff
@@ -85,7 +85,15 @@ Important points you should consider when training your networks with compressio
   - Turn off the `Dropout` layers (and similar ones like `DropConnect`) when training a network with quantization or sparsity
   - It is better to turn off additional regularization in the loss function (for example, L2 regularization via `weight_decay`) when training the network with RB sparsity, since it already imposes an L0 regularization term.
 
-#### Step 4 (optional): Export the compressed model to ONNX
+#### Step 4 (optional): Prepare for inference model.
```

Comment:
I'm not sure but need to show scenario of using: `If you need to ... do the following ...`